cluster/engine.go

```diff
@@ -385,6 +385,11 @@ func (e *Engine) UpdatedAt() time.Time {
 	return e.updatedAt
 }
 
+// emit engine_connected event removeDuplicateContainers
+func (e *Engine) AfterEngineValidationComplete() {
+	e.emitEvent("engine_connected")
```

Comment:
There is already `engined_connect` event at https://github.com/docker/swarm/blob/master/cluster/engine.go#L233. I think we should use that event.